src/common/components/fix-instruction-processor.tsx

```diff
@@ -89,6 +171,99 @@ export class FixInstructionProcessor {
         );
     }
 
+    private bulletAndColorBoxOutput(
+        sortedMatches: ColorMatch[],
+        fixInstruction: string,
+        insertionIndex: number,
+        result: JSX.Element[],
+        keyIndex: number,
+    ) {
+        for (let i: number = 0; i < sortedMatches.length; i++) {
+            const match: ColorMatch = sortedMatches[i];
+            if (i === 0) {
+                insertionIndex = 0;
+            } else {
+                insertionIndex =
+                    match.splitIndex -
+                    this.foregroundRecommendedColorText.length -
+                    match.colorHexValue.length -
+                    1;
+                if (i > 2) {
+                    insertionIndex -= 3;
+                }
+            }
+            let colorEndIndex = match.splitIndex - match.colorHexValue.length;
+            const beforeColorBox = fixInstruction.substring(insertionIndex, colorEndIndex);
+            insertionIndex = colorEndIndex;
+            i++;
+            colorEndIndex = sortedMatches[i].splitIndex - match.colorHexValue.length;
+            const middleSubstring = fixInstruction.substring(insertionIndex, colorEndIndex);
+            insertionIndex = colorEndIndex;
+            let endSubstring = '';
+            if (i === 1) {
+                endSubstring = fixInstruction.substring(
+                    insertionIndex,
+                    fixInstruction.indexOf(this.contrastRatioText) +
+                        this.contrastRatioText.length +
+                        6,
+                );
+                result.push(
+                    <span key={`instruction-split-${keyIndex++}`}>
+                        {this.addBulletPoint(
+                            keyIndex,
+                            beforeColorBox,
+                            match,
+                            middleSubstring,
+                            sortedMatches,
+                            i,
+                            endSubstring,
+                        )}
+                    </span>,
+                );
+            } else {
+                colorEndIndex = sortedMatches[i].splitIndex + this.recommendEndSentence.length;
+                endSubstring = fixInstruction.substring(insertionIndex, colorEndIndex);
+                result.push(
+                    <span key={`instruction-split-${keyIndex++}`}>
+                        <ul key={`instruction-split-${keyIndex++}`}>
```

Comment:
I gave the changes a quick pass in NVDA, and it looks like there is a slight accessibility issue with the new rows. Specifically, the way the list is now updated, NVDA always reads it as having one item, even when it now has more. NVDA read the following to me as I navigated through the list: "...Fix the following: list  with 1 items  •   Element has insufficient color contrast of 4.07 (foreground color: #0074d9, background color: #e6f5d0, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1 Use foreground color: #0068c3 and the original background color: #e6f5d0 to meet a contrast ratio of 4.87:1. Or use background color: #fdffe4 and the original foreground color: #0074d9 to meet a contrast ratio of 4.58:1...". Instead, I would expect NVDA to announce that I am entering a new list with two items before reading the suggested fixes.

Part of the issue is that each new `<li>` is being added in its own new `<ul>`--it would be more correct to create a new `<ul>` and then add the one or two suggested color `<li>` elements to that list. 

The other part of the issue relates to lines 168-9 of this file where we add separate screen reader text. I think we might be best served by removing that text, so I am going to approve this PR so we can deal with this in a follow-up PR if you would prefer (I could also do the follow up, given the timing of things). Please let me know if you would like to try fixing it before this PR is merged or not 🙂